vnext/Microsoft.ReactNative/Modules/DeviceInfoModule.cpp

```diff
@@ -102,19 +131,24 @@ void DeviceInfoHolder::updateDeviceInfo() noexcept {
 
     m_windowWidth = window.Bounds().Width;
     m_windowHeight = window.Bounds().Height;
-  } else {
-    /// TODO: WinUI 3 Island - mock for now
-    m_windowWidth = 600;
-    m_windowHeight = 800;
+  } else if (auto hwnd = (HWND)XamlUIService::GetIslandWindow(m_context->Properties())) {
+    RECT rect{};
+    if (CALL_INDIRECT(L"user32.dll", GetWindowRect, hwnd, &rect)) {
```

Comment:
>CALL_INDIRECT [](start = 8, length = 13)

I wonder why we cannot call the GetWindowRect directly.

Reply:
some APIs are not available to UWP apps